src/app/external-cluster-wizard/steps/external-cluster/provider/aks/style.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'variables';
+
+.edit-cluster-form {
+  margin-bottom: 30px;
+}
+
+km-label-form:not(.pod-node-selector-config) {
+  margin-top: 30px;
+}
+
+mat-checkbox {
+  i {
+    margin: 4px 8px;
+  }
+}
+
+mat-button-toggle-group[group='auditLogging'] {
```

Comment:
Why do we need this style? We're not using `mat-button-toggle-group` in `template.html`